msal/tests/Test.MSAL.NET.Integration/Test.MSAL.NET.Integration.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net4.5.2</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.8.0" />
+    <PackageReference Include="MSTest.TestAdapter" Version="1.3.2" />
+    <PackageReference Include="MSTest.TestFramework" Version="1.3.2" />
+    <PackageReference Include="Microsoft.Azure.KeyVault" Version="2.0.6" />
+    <PackageReference Include="Microsoft.Extensions.Configuration" Version="1.1.1" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="1.1.1" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.FileExtensions" Version="1.1.1" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="1.1.1" />
+    <PackageReference Include="Microsoft.Identity.Labs" Version="2.1.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="10.0.2" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\core\tests\Test.Microsoft.Identity.Core.UIAutomation\Test.Microsoft.Identity.Core.UIAutomation.csproj" />
```

Comment:
>ude="..\..\..\cor [](start = 26, length = 17)

why is this reference needed?

Reply:
Instead of having duplicate lab files, i just referenced the ones already in the UIAutomation project. We could also move them elsewhere for the different tests to target. What do you suggest?

---
In reply to: [220424544](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1271#discussion_r220424544) [](ancestors = 220424544)

Reply:
having one location with these in that all other solutions include would be my preference. it can be linked files, it doesn't have to be a project on it's own.

Reply:
sounds good. 

---
In reply to: [220428564](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1271#discussion_r220428564) [](ancestors = 220428564)

Reply:
Bogdan and I discussed this and he suggested making a separate project in core to hold these files. I went that route, but if you'd prefer linked files, we can do that. 

---
In reply to: [220428738](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1271#discussion_r220428738) [](ancestors = 220428738,220428564)